packages-proprietary/tokens/src/components/ams/button.tokens.json

```diff
@@ -13,41 +13,42 @@
       "disabled": {
         "cursor": { "value": "{ams.cursor.disabled}" }
       },
-      "forced-color-mode": {
-        "border": { "value": "{ams.border.width.m} solid" }
-      },
       "primary": {
         "background-color": { "value": "{ams.color.interactive.default}" },
-        "box-shadow": { "value": "inset 0 0 0 {ams.border.width.m} {ams.color.interactive.default}" },
+        "border": { "value": "{ams.border.width.m} solid {ams.color.interactive.default}" },
         "color": { "value": "{ams.color.text.inverse}" },
         "disabled": {
           "background-color": { "value": "{ams.color.interactive.disabled}" },
-          "box-shadow": { "value": "inset 0 0 0 {ams.border.width.m} {ams.color.interactive.disabled}" }
+          "border": { "value": "{ams.border.width.m} solid {ams.color.interactive.disabled}" }
         },
         "hover": {
           "background-color": { "value": "{ams.color.interactive.hover}" },
-          "box-shadow": { "value": "inset 0 0 0 {ams.border.width.m} {ams.color.interactive.hover}" }
+          "border": { "value": "{ams.border.width.m} solid {ams.color.interactive.hover}" },
+          "box-shadow": { "value": "inset 0 0 0 {ams.border.width.s} {ams.color.interactive.hover}" }
```

Comment:
I don't think our primary button needs this?

Reply:
It does need a border as it influences the size of the button. Removed the box shadow.